### DISCUSSION TITLE
.pkg installer doesn't give PATH commands

### DISCUSSION THREAD

**Discussion by DanMarcusPactFi** [Homebrew/discussions#4689] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.3
ORIGIN: https://github.com/Homebrew/brew
HEAD: 128df14e0afd0525564051a7665708c2973a3cab
Last commit: 27 hours ago
Core tap: N/A
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

This is more feedback now that I've resolved the issue, but it's a really dumb issue to start with.

After using the .pkg installer provided (https://github.com/Homebrew/brew), the final steps read as follows:
"Next steps:
To add Homebrew to your PATH run brew shellenv in your shell profile (e.g. ~/.bash_profile or ~/.zprofile)
Run brew help to get started"

How the heck is anyone supposed to know what "run brew shellenv in your shell profile (e.g. ~/.bash_profile or ~/.zprofile)" means?
Running `brew help` doesn't actually _DO_ anything yet, as the above step is required to set the PATH.

By comparison, installing via CLI provides the commands:
```
Next steps:
- Run these two commands in your terminal to add Homebrew to your **PATH**:
echo 'eval "$(/opt/homebrew/bin/brew shellenv)" ' >> /Users/user/.profile
eval "$(/opt/homebrew/bin/brew shellenv)"
```

Why would these steps be described for the curl install but not for the .pkg install which is supposed to make things easier?

**Comment by osalbahr** [comment]
> Why would these steps be described for the curl install but not for the .pkg install which is supposed to make things easier?

It shouldn’t. I think the same issue was reported in the following issue:

- https://github.com/Homebrew/brew/issues/15793

And PR:

- https://github.com/Homebrew/brew/pull/15794

The fix will probably be in of the next few releases.

**Comment by tommipontinen76** [comment]
How to do this on linux?
echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile returns an error because the file doesnt exist

**Reply by osalbahr** [reply]
On Linux, the eval command should use `/home/linuxbrew/.linuxbrew/bin/brew` instead. The install script should already instruct to do so at the end.
